File: src/main/java/phoenix/client/models/entity/CaudaEyesLayer.java
package phoenix.client.models.entity;

import net.minecraft.client.renderer.RenderType;
import net.minecraft.client.renderer.entity.IEntityRenderer;
import net.minecraft.client.renderer.entity.layers.AbstractEyesLayer;
import net.minecraft.entity.LivingEntity;
import net.minecraft.util.ResourceLocation;
import org.jetbrains.annotations.NotNull;

public class CaudaEyesLayer<T extends LivingEntity> extends AbstractEyesLayer<T, Cauda2Model<T>> {
    private static final RenderType RENDER_TYPE = RenderType.getEyes(new ResourceLocation("phoenix", "textures/entity/cauda/cauda_eyes.png"));

    public CaudaEyesLayer(IEntityRenderer<T, Cauda2Model<T>> rendererIn) {
        super(rendererIn);
    }

    @NotNull
    public RenderType getRenderType() {
        return RENDER_TYPE;
    }
}

File: src/main/kotlin/phoenix/client/models/entity/CaudaEyesLayer.kt
package phoenix.client.models.entity

import net.minecraft.client.renderer.RenderType
import net.minecraft.client.renderer.entity.IEntityRenderer
import net.minecraft.client.renderer.entity.layers.AbstractEyesLayer
import net.minecraft.entity.LivingEntity
import net.minecraft.util.ResourceLocation

class CaudaEyesLayer<T : LivingEntity>(rendererIn: IEntityRenderer<T, Cauda2Model<T>>) : AbstractEyesLayer<T, Cauda2Model<T>>(rendererIn)
{
    override fun getRenderType(): RenderType
    {
        return RENDER_TYPE
    }

    companion object
    {
        private val RENDER_TYPE = RenderType.getEyes(ResourceLocation("phoenix", "textures/entity/cauda/cauda_eyes.png"))
    }
}